azure-pipelines.yml

```
@@ -90,9 +90,9 @@ variables:
 
   - name: HelixWindowsVsQueueName
     ${{ if eq(variables['System.TeamProject'], 'public') }}:
-      value: windows.amd64.vs2022.pre.open
+      value: windows.amd64.vs2026.pre.open
```

Based on https://helix.dot.net/
```suggestion
      value: windows.amd64.vs2026.pre.scout.open
```